neuro_config_client/client.py

```diff
@@ -276,6 +279,159 @@ async def remove_storage(
                 raise
         return await self.get_cluster(cluster_name)
 
+    async def get_node_pool(
+        self,
+        cluster_name: str,
+        node_pool_name: str,
+        *,
+        token: str | None = None,
+    ) -> NodePool:
+        assert self._client
+        url = (
+            self._clusters_url
+            / cluster_name
+            / "cloud_prodider"
+            / "node_pools"
+            / node_pool_name
+        )
+
+        headers = self._create_headers(token=token)
+        async with self._client.get(url=url, headers=headers) as responce:
+            responce.raise_for_status()
+            resp_payload = await responce.json()
+            return self._entity_factory.create_node_pool(resp_payload)
+
+    async def get_node_pools(
+        self,
+        cluster_name: str,
+        *,
+        token: str | None = None,
+    ) -> list[NodePool]:
+        assert self._client
+        url = self._clusters_url / cluster_name / "cloud_prodider/node_pools"
+
+        headers = self._create_headers(token=token)
+        async with self._client.get(url=url, headers=headers) as responce:
+            responce.raise_for_status()
+            resp_payload = await responce.json()
+            return [self._entity_factory.create_node_pool(n) for n in resp_payload]
+
+    async def add_node_pool(
+        self,
+        cluster_name: str,
+        node_pool: NodePool,
+        *,
+        token: str | None = None,
+        start_deployment: bool = True,
+    ) -> Cluster:
```

Comment:
In the future, we might need to change the API so it returns added/patched/deleted node_pool. WDYT?

Reply:
When you change node pools other parts of cluster also change, so it's better to return the whole cluster. The same behavior for storages.